src/components/messages/MessageBox/index.tsx

```diff
@@ -47,19 +58,48 @@ const MessageBox: React.FC<MessageBoxProps> = ({ topic, authorAccount }) => {
 
   return (
     <div className="MessageBox">
-      <Textarea
-        placeholder="Message..."
-        value={messageText}
-        onChange={({ target }) => setMessageText(target.value)}
-      />
-      <button
-        onClick={onSend}
-        title={messageText === '' ? 'Message cannot be empty' : 'Send message'}
-        disabled={messageText === ''}
-        className="MessageBox__send"
-      >
-        <SendIcon />
-      </button>
+      {isDisabled ? (
+        <p className="MessageBox__status">
+          {inviteStatus === 'pending' ? (
+            <>Waiting to accept your invite…</>
+          ) : (
+            <>
+              Your chat invite was declined{' '}
+              <button className="MessageBox__status--btn">
```

Comment:
My bad, I should've given you a heads up but we're removing the retry button. :pray:  Please remove it

Reply:
Ah! No problem :)